.github/workflows/ci.yml

```
@@ -27,75 +41,171 @@ jobs:
         uses: subosito/flutter-action@v2
         with:
           channel: "stable"
+          flutter-version: 3.32
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)